roles/varnish/templates/etc/varnish/varnish.vcl

```diff
@@ -160,6 +170,11 @@ sub vcl_recv {
         return (pass);
     }
 
+    if (req.url ~ "^/api/") {
```

Comment:
So, press is claiming cnx.org/api ? That's a pretty serious land-grab for something that has been described as temporary. Are we rethinking the likely lifespan of press? Do we intend it to take over everything publishing is currently doing?

Also, so far we've kept cnx.org for humans to interact with, archive.cnx.org as the machine interface. Would that be a better choice for where press's API lives, whatever it's route?

Reply:
It's the lowest common match item at the moment. Things can always change.

We can put it on archive.cnx.org if that feels better for you. I'd like to see everything under one domain, but it's apples and oranges. 

I'm personally seeing press as something more long-term now. But this is a discussion to have outside the scope of this PR.